packages/apify/src/apify.ts

```diff
@@ -823,26 +811,19 @@ export class Apify {
      * NPM package, and it is automatically configured using the `APIFY_API_BASE_URL`, and `APIFY_TOKEN`
      * environment variables. You can override the token via the available options. That's useful
      * if you want to use the client as a different Apify user than the SDK internals are using.
-     *
-     * @param {object} [options]
-     * @param {string} [options.token]
-     * @param {string} [options.maxRetries]
-     * @param {string} [options.minDelayBetweenRetriesMillis]
      */
-    newClient(options = {}): ApifyClient {
+    newClient(options: Omit<ApifyClientOptions, 'baseUrl' | 'requestInterceptors' | 'timeoutSecs'> = {}): ApifyClient {
         return this.config.createClient(options);
     }
 
     /**
      * Returns `true` when code is running on Apify platform and `false` otherwise (for example locally).
-     *
-     * @returns {boolean}
      */
-    isAtHome() {
+    isAtHome(): boolean {
         return !!this.config.get('isAtHome');
     }
 
-    get utils() {
+    get utils(): Record<string, unknown> {
```

Comment:
this needs to be properly typed, we need to know the exact keys as well as their values

on the other hand iirc we are planning to ditch this static utils register, so maybe its not worth it? cc @mnmkng

Reply:
Yeah, let's ignore it for now. I'm really looking forward to having the utils available directly from the `crawlingContext`. E.g.

```js
function handlePageFunction(ctx) {
    // ...
    await ctx.enqueueLinks({ selector: 'a.item' })
}
```

Instead of the current:

```js
function handlePageFunction(ctx) {
    // ...
    await Apify.utils.enqueueLinks({
        page: ctx.page,
        requestQueue: ctx.crawler.requestQueue,
        selector: 'a.item'
    })
}
```